### DISCUSSION TITLE
Stack Processing Upgrade

### DISCUSSION THREAD

**Discussion by VonZato** [TechReborn/TechReborn#3056] [Ideas]
I think adding the ability for a stack processing upgrade (Being able to process one stack at a time) Would be greatly appreciated, I would rather have one machine able to work slower with just 3 overclockers that can process a whole stack instead of what I have now which is like 20 machines going at full speed, it would also cut down on lag. Also, just make it stupid expensive I would still make it regardless.

**Comment by Ayutac** [comment]
As an upgrade, it is problematic. Because some recipes don't do 1:1 processing. Can't do 64 items if 128 come out. And TR will most likely not screw around with stack sizes.

But maybe we could do higher tier machines that do something to this effect.
